src/app/content/components/LabsCall.tsx

```diff
@@ -160,7 +160,7 @@ const LabsCTA = () => {
     <LabsCallWrapper data-async-content>
       <LabsCallHeader>
         <Column noShrink>
-          <LabsLogo src={icon} role='img' alt='' />
```

Comment:
`The ARIA img role can be used to identify multiple elements inside page content that should be considered as a single image.`
The element is a single `<img>` tag. It does not need the `img` role.